node-fs-writefile/cp.js

```diff
@@ -0,0 +1,16 @@
+import { readFile } from 'node:fs/promises';
+import { writeFile } from 'node:fs/promises';
+
+const srcPath= `./${process.argv[2]}`;
+const destPath = `./${process.argv[3]}`;
```

Comment:
The leading `./` here isn't necessary since readFile and writeFile will default to looking in the current directory.